#------------- Environment parameters ----------------
#
# Defines parameters external to the rocket:
#   - Athmosphere parameters
#   - Rail configuration
#   - Targeted apogee
#
# ----------------------------------------------------
ground_altitude: 1567
latitude: 46.90479
longitude: 8.07575
temperature: 300
humidity: 0.5
pressure: 84972.484
apogee: # targeted apogee
  - 0
  - 0
  - 3048
wind_speed: 8
wind_direction: -90 # angle of wind from north, trigonometric
rail_azimuth: 0 # angle of rail from north, trigonometric
rail_zenith: 18 # angle of rail from vertical (trigonometric inertial Y rotation)
rocket_roll: -135 # initial roll angle around z rocket axis
rail_length: 4
